src/Line.php

```diff
@@ -135,6 +147,50 @@ public function isFirst()
         return $this->previous() === false;
     }
 
+    /**
+     * Get the Lexer
+     * 
+     * @since 1.2.0
+     * 
+     * @return Lexer
+     */
+    public function getLexer()
+    {
+        return $this->lexer;
+    }
+
+    /**
+     * Get the line's input in a safe way.
+     * 
+     * Escaping for html is done if this wasn't done by a previous listener already.
+     * 
+     * @since 1.2.0
+     * 
+     * @return string
+     */
+    public function getInput()
+    {
+        if ($this->isEscaped()) {
+            return $this->input;
+        }
+        
+        return $this->lexer->escape($this->getUnsafeInput());
+    }
+
+    /**
+     * Get the raw line's input, this might not be escaped for html context.
+     * 
+     * Note it could be escaped if a previous inline listener updated the input value
```

Comment:
If you like to make hints more clear you can use 

```php
/*
 * XYZ
 * 
 * > HINT XYZ
 */
```

As if we will use a phpdoc renderer, this will be markdown quote. But not necessary just a general info.

Reply:
Good one, thanks! Never worked with that automated rendering yet.